website/content/partials/telemetry-metrics/vault/route/rollback/mountpoint.mdx

```diff
@@ -5,4 +5,15 @@ Metric type | Value | Description
 summary     | ms    | Time required to send a rollback request to the backend and for the backend to complete the operation for the given mount point
 
 Vault automatically schedules and performs mount point rollback operations to
-clean up partial errors.
\ No newline at end of file
+clean up partial errors.
+
+### vault.route.rollback ((#vault-route-rollback))
```

Comment:
The new metrics should have their own partials and the relevant metric indexes should be updated to include them.